examples/mixture_gaussian.py

```diff
@@ -98,24 +98,38 @@ def log_prob(self, xs, zs):
 
         return log_prior + tf.pack(log_lik)
 
+    def predict(self, xs, zs):
+        """Return matrix with log-likelihoods for each data point under each cluster, 
+        averaging over each set of latent variables z in zs."""
+        x = xs['x']
+        pi, mus, sigmas = zs
+        pi = tf.reduce_mean(pi, 0)
+        mus = tf.reduce_mean(mus, 0)
+        sigmas = tf.reduce_mean(sigmas, 0)
+
+        matrix = []
+        for k in range(self.K):
+            matrix += [multivariate_normal.logpdf(x,
+                               mus[(k*self.D):((k+1)*self.D)],
+                               sigmas[(k*self.D):((k+1)*self.D)])]
+
+        return tf.pack(matrix)
```

Comment:
For PEP 8 reasons, can you add an additional blank line here after this return statement? That is, [surround top-level function and class definitions with two blank lines.](https://www.python.org/dev/peps/pep-0008/#blank-lines)